scripts/package.json

```diff
@@ -11,9 +11,9 @@
     "@fluentui/eslint-plugin": "*"
   },
   "dependencies": {
-    "@mdx-js/loader": "^1.5.5",
+    "@mdx-js/loader": "^1.6.22",
     "@microsoft/load-themed-styles": "^1.10.26",
-    "@microsoft/loader-load-themed-styles": "^1.7.205",
+    "@microsoft/loader-load-themed-styles": "^2.0.17",
```

Comment:
this package doesn't follow semver. in version 1.9.x they bumped load-theme-styles to 2.0. in 2.0. they moved to use peerDeps for load-theme-styles. nevertheless the major bump doesn't affect functionality and everything should work as expected